addons/account/models/account_invoice.py

```diff
@@ -1150,6 +1150,8 @@ def _get_tax_amount_by_group(self):
         ) for r in res]
         return res
 
+    def _map_tax_partner(self):
```

Comment:
@nim-odoo 
I wouldn't provide such hook, as this opens the door for inconsistencies if not applied on every single model.
In order to determine the context of my transaction, I think custom overrides of _map_tax on fp are pretty much sufficient.

What case where you thinking of?